client.js

```diff
@@ -45,100 +42,338 @@ bot.on('message', function (message) {
+                                            user.send(`Rise and grind.  Your account is no longer snoozed.`).catch(console.error);
+                                            break;
+                                        default:
+                                            break;
+                                    }
+                                })
+                            })
+                        })
+                        .catch(console.error);
                     }  else {
-                        message.author.send(`You don\'t have any watches.`);
+                        message.author.send("You don\'t have any watches.  Add some with `!watch`");
                     }
                 });
                 break;
 
-            // !end all watches
-            case 'END ALL WATCHES':
-                // console.log('end all watches command received.  args = ', args)
-                db.endAllWatches(message.author.id);
-                message.author.send('All watches succesfully ended.');
+            // !extend
+            case 'EXTEND':
+                message.author.send('All watches succesfully extended for another 7 days.');
                 break;
 
-            // !extend all watches
-            case 'EXTEND ALL WATCHES':
-                // console.log('extend all watches command received.  args = ', args)
-                db.extendAllWatches(message.author.id);
-                message.author.send('All watches succesfully extended for another 7 days.');
+            // !add block
+            case 'BLOCK':
+                if (args[1] === 'BLUE' || args[1] === 'GREEN') {
+                    db.blockSeller(message.author.id, args[0], args[1], null)
+                    message.author.send(`Lets cut down the noise.  No longer notifying you about auctions from ${args[0]} on ${args[1]} for any current or future watches.`);
+                } else {
+                    db.blockSeller(message.author.id, args[0], null, null)
+                    message.author.send(`Lets cut down the noise.  No longer notifying you about auctions from ${args[0]} on both servers for any current or future watches.  To only block this seller on one server, use \`\`!add block: ${args[0]}, server\`\``);
+                }
+                break;
+
+            //TODO:
+            case 'UNBLOCK':
+                if (args[1] === 'BLUE' || args[1] === 'GREEN') {
+                    db.unblockSeller(message.author.id, args[0], args[1], null)
+                    message.author.send(`People change.  No longer blocking ${formatCapitalCase(args[0])} on ${formatCapitalCase(args[1])} server.`)
+                } else {
+                    db.unblockSeller(message.author.id, args[0], null, null)
+                    message.author.send(`People change.  No longer blocking ${formatCapitalCase(args[0])} on either server.`)
+                }
+                break;
+
+            //TODO:
+            case 'BLOCKS':
```

Comment:
Does the output of this command allow you to un-do the blocks in any way? I'm not clear how this would work, as it looks like right now you're building a single big message?
Should it respond with one message per block, and have an emoji to press to undo that block?

Reply:
That's a great idea, let's add it to Issues as a feature to do.